docs/download_data.rst

```diff
@@ -0,0 +1,39 @@
+Data Access
+===========
+
+The electrophysiology data files for the PatchSeq experiments released by the
+Allen Institute are stored in `Neurodata Without Borders 2.0 <https://nwb.org>`_ (NWB) format.
+The files are hosted on the `Distributed Archives for Neurophysiology Data Integration (DANDI) <https://dandiarchive.org>`_.
+
+The PatchSeq data release is composed of  two archives:
+
+Mouse data archive (114 GB): `<https://dandiarchive.org/dandiset/000020>`_
+
+Human data archive (12 GB): `<https://dandiarchive.org/dandiset/000023>`_
+
+You can download these data following the above links or
+using DANDI's command line client that can be installed as:
+
+.. code-block:: bash
+
+    pip install dandi
+
+With the client installed, you can easily download individual files or an entire archive as:
+
+.. code-block:: bash
+
+    dandi download --output-dir <DIRECTORY> <URL>
+
+where <DIRECTORY> is the existing directory on your file system
+and <URL> is the url path of a file or archive.
+
+The paths to individual data files withing the archive are listed in the file manifest
+(need links) under the ``arhive_uri`` column.
```

Comment:
"archive_uri" typo

Reply:
also didn't we get these file manifests last week?

Reply:
per discussion with @wbwakeman, the manifests will be posted on a "Explore" page on brain-map.org. When this page is live we should be able to link to it.